src/qt/rpcconsole.cpp

```diff
@@ -462,15 +462,17 @@ RPCConsole::RPCConsole(interfaces::Node& node, const PlatformStyle *_platformSty
 
     constexpr QChar nonbreaking_hyphen(8209);
     const std::vector<QString> CONNECTION_TYPE_DOC{
-        tr("Inbound: initiated by peer"),
-        tr("Outbound Full Relay: default"),
-        tr("Outbound Block Relay: does not relay transactions or addresses"),
-        tr("Outbound Manual: added using RPC %1 or %2/%3 configuration options")
+        //: The connection direction (Inbound/Outbound) is also used in ConnectionTypeToQString() and PeerTableModel::data().
+        //: The connection types (Full Relay, Block Relay, Manual, Feeler, Address Fetch) are also used in ConnectionTypeToQString().
+        tr("Inbound") + ": " + tr("initiated by peer"),
```

Comment:
nit: Now there's lot's of string concatenations, maybe worth using QStringBuilder's `%` and wrapping " " and ": " with `QLatin1String` ?

Reply:
Thanks @Talkless. Added QLatin1String constants in 73cf0fdc8edacf9294f9facc8248fb078298b2d7. Let me know if you think using `%` would still improve the diff and what you'd suggest.